HHS/OPRE-OPS
# (5) feat: CanReviewAccordion improvements on Agreement Approve page

## What changed

- `AgreementCANReviewAccordion`
- and child component `CANFundingCard`
- UX Review items for toggle actions
- `Obligate By Date` from `Date Needed` label change

## Issue

#2349 #2630 

## How to test

1. Make a Budget Change on Agreement 9
2. Send to Approval
3. As an approver, see the Approve Agreement page
4. Toggle before/after approval
5. See CANFundingCards account for the change(s)

## Screenshots

https://github.com/user-attachments/assets/60528d93-9aeb-4d80-a193-b1fa7079fc84

https://github.com/user-attachments/assets/08fe5a78-ac16-43bc-a8e4-5a0a32c11cef

https://github.com/user-attachments/assets/1ba39206-37af-4239-b310-9bab0754880a

## Definition of Done Checklist


## Links

Links to prior PRs:
1. https://github.com/HHS/OPRE-OPS/pull/2516
1. https://github.com/HHS/OPRE-OPS/pull/2544
1. https://github.com/HHS/OPRE-OPS/pull/2554
1. https://github.com/HHS/OPRE-OPS/pull/2558
1. https://github.com/HHS/OPRE-OPS/pull/2570
1. https://github.com/HHS/OPRE-OPS/pull/2585
